8.11.0: pytest requires `conftest`

Looks like pytest requires `conftest`
```console
+ /usr/bin/pytest -ra -m 'not network' --import-mode=importlib
============================= test session starts ==============================
platform linux -- Python 3.8.18, pytest-7.4.3, pluggy-1.3.0
rootdir: /home/tkloczko/rpmbuild/BUILD/pikepdf-8.11.0
configfile: pyproject.toml
testpaths: tests
plugins: hypothesis-6.84.3, timeout-2.2.0
collected 441 items / 4 errors / 1 skipped

==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_errors.py _____________________
ImportError while importing test module '/home/tkloczko/rpmbuild/BUILD/pikepdf-8.11.0/tests/test_errors.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_errors.py:7: in <module>
    from conftest import skip_if_pypy
E   ModuleNotFoundError: No module named 'conftest'
____________________ ERROR collecting tests/test_ipython.py ____________________
ImportError while importing test module '/home/tkloczko/rpmbuild/BUILD/pikepdf-8.11.0/tests/test_ipython.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_ipython.py:14: in <module>
    from conftest import fails_if_no_mutool
E   ModuleNotFoundError: No module named 'conftest'
___________________ ERROR collecting tests/test_metadata.py ____________________
ImportError while importing test module '/home/tkloczko/rpmbuild/BUILD/pikepdf-8.11.0/tests/test_metadata.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_metadata.py:13: in <module>
    from conftest import skip_if_ci, skip_if_pypy
E   ModuleNotFoundError: No module named 'conftest'
____________________ ERROR collecting tests/test_object.py _____________________
ImportError while importing test module '/home/tkloczko/rpmbuild/BUILD/pikepdf-8.11.0/tests/test_object.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_object.py:14: in <module>
    from conftest import skip_if_pypy
E   ModuleNotFoundError: No module named 'conftest'
=========================== short test summary info ============================
SKIPPED [1] tests/test_io_psutil.py:15: could not import 'psutil': No module named 'psutil'
ERROR tests/test_errors.py
ERROR tests/test_ipython.py
ERROR tests/test_metadata.py
ERROR tests/test_object.py
!!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 skipped, 4 errors in 1.62s =========================
```
I cannot find that module
```console
$ pip download conftest
ERROR: Could not find a version that satisfies the requirement conftest (from versions: none)
ERROR: No matching distribution found for conftest
```
🤔

--- a/src/pikepdf/__init__.py
+++ b/src/pikepdf/__init__.py
@@ -7,15 +7,15 @@
 
 from __future__ import annotations
 
-from ._version import __version__
+from pikepdf._version import __version__
 
 try:
-    from . import _core
+    from pikepdf import _core
 except ImportError as _e:  # pragma: no cover
     _msg = "pikepdf's extension library failed to import"
     raise ImportError(_msg) from _e
 
-from ._core import (
+from pikepdf._core import (
     AccessMode,
     Annotation,
     AttachedFileSpec,
@@ -41,8 +41,8 @@ from ._core import (
     TokenFilter,
     TokenType,
 )
-from ._exceptions import DependencyError
-from .objects import (
+from pikepdf._exceptions import DependencyError
+from pikepdf.objects import (
     Array,
     Dictionary,
     Name,
@@ -52,7 +52,7 @@ from .objects import (
     Stream,
     String,
 )
-from .models import (
+from pikepdf.models import (
     Encryption,
     Outline,
     OutlineItem,
@@ -73,8 +73,8 @@ from .models import (
 # While _cpphelpers is intended to be called from our C++ code only, explicitly
 # importing helps introspection tools like PyInstaller figure out that the module
 # is necessary.
-from . import _cpphelpers, _methods, codec  # noqa: F401, F841
-from . import settings
+from pikepdf import _cpphelpers, _methods, codec  # noqa: F401, F841
+from pikepdf import settings
 
 __libqpdf_version__: str = _core.qpdf_version()
 

--- a/src/pikepdf/codec.py
+++ b/src/pikepdf/codec.py
@@ -8,7 +8,7 @@ from __future__ import annotations
 import codecs
 from typing import Any, Container
 
-from ._core import pdf_doc_to_utf8, utf8_to_pdf_doc
+from pikepdf._core import pdf_doc_to_utf8, utf8_to_pdf_doc
 
 # pylint: disable=redefined-builtin
 

--- a/src/pikepdf/models/__init__.py
+++ b/src/pikepdf/models/__init__.py
@@ -5,18 +5,18 @@
 
 from __future__ import annotations
 
-from ._content_stream import (
+from pikepdf.models._content_stream import (
     ContentStreamInstructions,
     PdfParsingError,
     UnparseableContentStreamInstructions,
     parse_content_stream,
     unparse_content_stream,
 )
-from .encryption import Encryption, EncryptionInfo, Permissions
-from .image import PdfImage, PdfInlineImage, UnsupportedImageTypeError
-from .matrix import PdfMatrix
-from .metadata import PdfMetadata
-from .outlines import (
+from pikepdf.models.encryption import Encryption, EncryptionInfo, Permissions
+from pikepdf.models.image import PdfImage, PdfInlineImage, UnsupportedImageTypeError
+from pikepdf.models.matrix import PdfMatrix
+from pikepdf.models.metadata import PdfMetadata
+from pikepdf.models.outlines import (
     Outline,
     OutlineItem,
     OutlineStructureError,

--- a/src/pikepdf/objects.py
+++ b/src/pikepdf/objects.py
@@ -22,8 +22,8 @@ from __future__ import annotations
 from secrets import token_urlsafe
 from typing import TYPE_CHECKING, Any, Iterable, Mapping, cast
 
-from . import _core
-from ._core import Matrix, Object, ObjectType, Rectangle
+from pikepdf import _core
+from pikepdf._core import Matrix, Object, ObjectType, Rectangle
 
 if TYPE_CHECKING:  # pragma: no cover
     from pikepdf import Pdf

--- a/src/pikepdf/settings.py
+++ b/src/pikepdf/settings.py
@@ -5,7 +5,7 @@
 
 from __future__ import annotations
 
-from ._core import (
+from pikepdf._core import (
     get_decimal_precision,
     set_decimal_precision,
     set_flate_compression_level,

